libcontainer/container_linux.go

```diff
@@ -522,6 +522,27 @@ func (c *linuxContainer) commandTemplate(p *Process, childInitPipe *os.File, chi
 	return cmd
 }
 
+// sendMountSources says whether the child process must setup bind mounts with
+// the source pre-opened (O_PATH) in the host user namespace.
+// See https://github.com/opencontainers/runc/issues/2484
+func (c *linuxContainer) sendMountSources() bool {
```

Comment:
What about calling it `shouldSendMountSources()` or something similar that indicates it returns true/false without sending anything?